src/js/components/location/LocationChooser/index.jsx

```diff
@@ -76,8 +76,9 @@ const LocationChooser = (props) => {
     props.changeCurrentLocation(location).then(() => {
       props.fetchMenuConfig();
       props.fetchSessionInfo();
+
+      window.location.reload();
     });
-    setIsOpen(false);
```

Comment:
this is not the solution we would want to have. This way, you would always reload the page, it's not what React's flow is about (SPA). 
What you need to do is to refetch the data when the location changes (using appropriate `useEffect`)

Reply:
:point_up: